_meta/beat.yml

```diff
@@ -49,9 +49,27 @@ apm-server:
     # To enable real user monitoring (RUM) support set this to true.
     #enabled: false
 
+    #-- v1 RUM endpoint
+
     # Rate limit per second and IP address for requests sent to the RUM endpoint.
     #rate_limit: 10
 
+    #-- v2 RUM endpoint
+
+    #event_rate:
+      # Defines the maximum amount of events allowed to be sent to the APM Server v2 RUM
+      # endpoint per ip per second. There is an internal burst queue to avoid rejecting short termed spikes. The burst 
+      # queue is set to limit*5.
+      # Defaults to 1000.
+      #limit: 1000
+      # Defines the amount of unique IPs for which a rate limiter is collected.
+      # Ensure a large enough cache size, as the rate limiting cannot properly be applied,
+      # in case the amount of unique IPs exceeds the amount of cache keys configured.
+      # Defaults to 1000.
+      #cache_keys: 1000
```

Comment:
i like this generally, but the warning here could sound like rate limiting just stops working or something when the LRU size is exceeded. I think we can either:
* explain in detail how it works or what happens when the number of IPs exceed the cache size. Something like:
```
# When an IP gets evicted from the LRU, its rate limit is re-used for the new IP
# to ensure that the rate limit is effective even when the concurrent number of unique clients exceed the LRU size.
# Sites with many concurrent clients should consider increasing this limit.
```
* or remove the warning and just say something like:
```
# An LRU cache is used to keep a rate limit per IP for the most recently seen IPs. 
# This setting defines the number of unique IPs that can will be tracked in the cache.
# Sites with many concurrent clients should consider increasing this limit.
```

Reply:
Should we consider naming it `lru_size`? `cache_keys` here is a bit ambiguous. No strong feelings

Reply:
I prefer to remove the warning, as I would not want to go too much into implementation details in the config yml description.